README.md

```diff
@@ -157,7 +157,9 @@ ember exam --split=3 --partition=3
 
 **Note 1**: _Ember Exam will respect the `parallel` setting of your [Testem config file](https://github.com/testem/testem/blob/master/docs/config_file.md#config-level-options) while running tests in parallel. The default value for `parallel` in Testem is 1, which means you'll need a non-default value to actually see parallel behavior._
 
-**Note 2**: _You must be using Testem version `1.5.0` or greater for this feature to work properly._
+**Note 2**: _Ember Exam sets `process.env.EMBER_EXAM_SPLIT` for convenience. You can use this in your Testem file._
```

Comment:
I'd slightly prefer `EMBER_EXAM_SPLIT_COUNT`.  I find `EMBER_EXAM_SPLIT` _slightly_ confusing (is it the total number of splits, or the current partition number?)...

Reply:
@trentmwillis Thoughts on the matter?

Reply:
I think I prefer `EMBER_EXAM_SPLIT` because it maps well to `--split`, as well as a future `EMBER_EXAM_PARTITION` mapping to `--partition`, but I'm cool with whatever the consensus is.

Reply:
@kellyselden I agree with you. I can understand @rwjblue's point, but if we mirror other options in the future, I'd like a consistent naming pattern.